src/test/scala/uk/gov/hmrc/test/ui/pages/section1/SummarySection1Page.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmrc.test.ui.pages.section1
+
+import uk.gov.hmrc.test.ui.pages.base.BasePage
+import uk.gov.hmrc.test.ui.pages.section1.DetailKeys.{Mucr, Section1}
+
+object SummarySection1Page extends BasePage {
+
+  def backButtonHref: String = if (maybeDetail(Mucr).isDefined) EnterAMucrPage.path else LinkMucrPage.path
+
+  def path: String = "/declaration/summary-section/1"
+
+  def title: String = "Check your answers"
+
+  override protected def performActionsAndStore(values: String*): Unit = {
+    checkSectionSummary(Section1)
+    continue()
```

Comment:
Do you actually need `continue`? Did not we agree that `continue` would have called by the feature files?